### PR TITLE
Update the strategy argument to 'construct a ReadableStream'.

### PR DESCRIPTION
**Pull request by jyasskin**
To match https://github.com/whatwg/fetch/pull/781.

@ricea

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/ServiceWorker/pull/1338.html" title="Last updated on Aug 13, 2018, 2:50 AM GMT (fcd5787)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/ServiceWorker/1338/73f5551...jyasskin:fcd5787.html" title="Last updated on Aug 13, 2018, 2:50 AM GMT (fcd5787)">Diff</a>

### CODE REVIEW

**Comment by ricea** [comment]
lgtm

File: docs/index.bs [hunk 1]
```diff
@@ -1437,14 +1437,15 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
...
                                 1. When |promise| is fulfilled with an object whose <code>done</code> property is true, set |end-of-body| to true.
                                 1. When |promise| is fulfilled with a value that matches with neither of the above patterns, or |promise| is rejected, [=ReadableStream/error=] |newStream| with a <code>TypeError</code>.
                             1. Let |cancel| be an action that [=ReadableStream/cancels=] |response|'s [=response/body=]'s <a>stream</a> with |reader|.
-                            1. Let |newStream| be the result of [=ReadableStream/construct a ReadableStream object=] with |strategy|, |pull| and |cancel| in |targetRealm|.
+                            1. Let |newStream| be the result of [=ReadableStream/construct a ReadableStream object=] with |highWaterMark|, |sizeAlgorithm|, |pull| and |cancel| in |targetRealm|.
```
[the remark this answers is not in the document]
Comment (jyasskin) [reply docs/index.bs]: D,o,n,e. :)

**Comment by jungkees** [comment]
Thanks for the PR @jyasskin! Could you remove the html file from the change set? Now Travis CI builds the spec when it's merged. (I just changed the CONTRIBUTING.md document.)

**Comment by jyasskin** [comment]
@jungkees Done.

**Comment by jungkees** [comment]
Thanks!

**Pull request merged**